CONTRIBUTORS_QUICK_START.rst

```diff
@@ -50,7 +50,7 @@ Local machine development
 
 If you do not work with remote development environment, you need those prerequisites.
 
-1. Docker Community Edition (you can also use Colima, see instructions below)
+1. Container runtime: Docker Community Edition (recommended), Colima.
```

Comment:
No matter which one you use, the container _runtime_ is always Docker. I don’t think other runtimes work with Breeze right now.

Reply:
Correct. Runtime is docker. 

In our case what we use from Colima is the replacement of Docker Desktop VM management and integration (mounts, sockets and the like). But we actually still use docker runtime. What Colima does, it starts Lima VM behind the scenes automatically as needed and under the hood it still uses the same docker runtime. 

We do not use `containerd` runtime (that Colima also supports). For that we would have to use https://github.com/containerd/nerdctl to run the commands..

Those are subtle differences that the users are not even aware of but we should be more precise, I think, in terminology :D. The `docker` is term that has been used for everything - Docker Runtime, Docker CLI, Docker Engine, recently Docker Desktop  - all those are different and pretty much each of them is replaceable (but only Docker Desktop requires an alternative because only Docker Desktop has this - prohibitive for many companies - licence). 

We are in a process of unbundling of those meanings, and I think it's good to be precise.

Colima in our case is used to replace "Docker Desktop" part while all the other "docker" parts remain (but Runtime and Engine are in Colima-managed VM).

BTW. Funny thing is that Colima on Linux stil uses Docker Engine and Runtime - but they are installed in the Lima VM.  So in a way you get rid of the engine (but only on a Host).

Reply:
FWIW, last time I tried using containerd with breeze, Docker CLI is not main problem (Podman is actually close enough you can just change a few constants to make things work), but docker-compose. But that’s off-topic; the main point here is the terminology here needs to be fixed to not introduce confusion unnecessarily.